src/components/help/views/ReportSummaryView.tsx

```diff
@@ -16,7 +16,8 @@ export const ReportSummaryView: FC<{}> = props =>
         {
             case ReportType.BULLY:
             case ReportType.EMERGENCY:
-            case ReportType.ROOM: {
+            case ReportType.ROOM:
+            case ReportType.PHOTO: {
```

Comment:
No need to add it here, since photo reports should be handled using a separate packet, if we want to stick to flash functionality